### PR TITLE
alpha docker typo

### PR DESCRIPTION
**Pull request by stakelovelace**
fixed

### CODE REVIEW

**Review by TrevorBenson — CHANGES_REQUESTED** [review]
Can you rebase from origin/alpha? That way everything accepted in #675 is filtered out of the PR for #676.

Thanks!

File: files/docker/node/alpha/dockerfile_stage3alpha [hunk 1]
```diff
@@ -107,7 +107,11 @@ ADD https://hydra.iohk.io/build/5102327/download/1/launchpad-db-sync-config.json
 ADD https://raw.githubusercontent.com/cardano-community/guild-operators/alpha/files/docker/node/addons/banner.txt /home/guild/.scripts/
 ADD https://raw.githubusercontent.com/cardano-community/guild-operators/alpha/files/docker/node/addons/master-topology.sh /home/guild/.scripts/
 ADD https://raw.githubusercontent.com/cardano-community/guild-operators/alpha/files/docker/node/addons/block_watcher.sh /home/guild/.scripts/
+<<<<<<< HEAD
```
Comment (TrevorBenson) [inline files/docker/node/alpha/dockerfile_stage3alpha:110]: Remove `<<<<<<< HEAD`

File: files/docker/node/alpha/dockerfile_stage3alpha [hunk 2]
```diff
@@ -107,7 +107,11 @@ ADD https://hydra.iohk.io/build/5102327/download/1/launchpad-db-sync-config.json
...
+ADD https://raw.githubusercontent.com/cardano-community/guild-operators/alpha/files/docker/node/addons/healthcheck.sh /home/guild/.scripts/
+=======
 ADD https://raw.githubusercontent.com/cardano-community/guild-operators/alpha/files/docker/nodlaze/addons/healthcheck.sh /home/guild/.scripts/
```
Comment (TrevorBenson) [inline files/docker/node/alpha/dockerfile_stage3alpha:113]: Remove the old nodlaze line

File: files/docker/node/alpha/dockerfile_stage3alpha [hunk 3]
```diff
@@ -107,7 +107,11 @@ ADD https://hydra.iohk.io/build/5102327/download/1/launchpad-db-sync-config.json
 ADD https://raw.githubusercontent.com/cardano-community/guild-operators/alpha/files/docker/node/addons/banner.txt /home/guild/.scripts/
 ADD https://raw.githubusercontent.com/cardano-community/guild-operators/alpha/files/docker/node/addons/master-topology.sh /home/guild/.scripts/
 ADD https://raw.githubusercontent.com/cardano-community/guild-operators/alpha/files/docker/node/addons/block_watcher.sh /home/guild/.scripts/
+<<<<<<< HEAD
+ADD https://raw.githubusercontent.com/cardano-community/guild-operators/alpha/files/docker/node/addons/healthcheck.sh /home/guild/.scripts/
+=======
```
Comment (TrevorBenson) [inline files/docker/node/alpha/dockerfile_stage3alpha:112]: Remove `=======`

File: files/docker/node/alpha/dockerfile_stage3alpha [hunk 4]
```diff
@@ -107,7 +107,11 @@ ADD https://hydra.iohk.io/build/5102327/download/1/launchpad-db-sync-config.json
...
 ADD https://raw.githubusercontent.com/cardano-community/guild-operators/alpha/files/docker/nodlaze/addons/healthcheck.sh /home/guild/.scripts/
+>>>>>>> origin/alpha
```
Comment (TrevorBenson) [inline files/docker/node/alpha/dockerfile_stage3alpha:114]: Remove `>>>>>>> origin/alpha`

**Review by TrevorBenson — COMMENTED** [review]
I believe you can update your local workstation branch by doing
```
git fetch -all
git pull
```
and this will show you the lines the merge resolution injected into origin/redoracle-docker branch. Then you can make the file look how you want and the commit will fix the PR.

File: files/docker/node/alpha/dockerfile_stage3alpha (hunk 3 above)
Comment (redoracle) [reply files/docker/node/alpha/dockerfile_stage3alpha:112]: Not sure what you are saying... can you explain?

File: files/docker/node/alpha/dockerfile_stage3alpha (hunk 4 above)
Comment (redoracle) [reply files/docker/node/alpha/dockerfile_stage3alpha:114]: Not sure what you are saying... can you explain?

Comment (TrevorBenson) [reply files/docker/node/alpha/dockerfile_stage3alpha:114]: This line `>>>>>>> origin/alpha` is now part of your `files/docker/node/alpha/dockerfile_stage3alpha` file inside the local github branch. You were to delete all content from the merge menu that you did not want and only leave the lines you did want. 

Unfortunately the way merge resolution looks is like it knows what you should do and would do it when you click to resolve. This is now how it works, instead it is all text put into your file and you need to edit the file to look how you want and then click resolve.

**Review by TrevorBenson — APPROVED** [review]
LGTM Merge resolution tags all removed.

**Pull request merged**